### PR TITLE
Added Section3.1 Business context

### PR DESCRIPTION
**Pull request by uo277776**
Hi @uo277915! I finished writing the business context section assigned in #15 . Any comment or suggestion to improve it, would be great.

I also send you here the image of the diagram, because you won't be able to see it unless you build the html file. You will only see the PlantUML code instead.

bcontext

### CODE REVIEW

**Review by uo277915 — CHANGES_REQUESTED** [review]
Hey, I have found some typos in the code but in general, I think you've done a really good job!

I especially like the diagram, it is very clear!

File: docs/03_system_scope_and_context.adoc [hunk 1]
```diff
@@ -23,23 +23,45 @@ Various options:
...
+[%header, cols=3]
+|===
+|Entity|Input|Output
+
+|User
+|User will receive the service provided by the app.
+|The user will create the pod and store there his/her information and also it will manage the app to ordewr products to buy.
+
+|WebApp
+|The app will display information store in the database and show it to the user. Also it will receive the information from the user using the PODs.
```
Comment (uo277915) [inline docs/03_system_scope_and_context.adoc:50]: I think the correct tense for 'store' is **stored**

File: docs/03_system_scope_and_context.adoc [hunk 2]
```diff
@@ -23,23 +23,45 @@ Various options:
...
+|It will make requests to the RestApi in order to get information about users, products, orders, etc. Also store the orders done by the users. 
+
+|RestApi
+|From the app, the api will receive requests to get information stored in the database and the database will return the result of performing queries.
+|The api will send queries to the database and convert the response received in order to send back the information in a proper way to the WebApp.
```
Comment (uo277915) [inline docs/03_system_scope_and_context.adoc:55]: I think this should end with "for the WebApp" instead of using 'to'

**Review by uo277915 — CHANGES_REQUESTED** [review]
I think I messed up and didn't add half the comments 🤡
Here are the rest of them!

--

Hey, I have found some typos in the code but in general, I think you've done a really good job!

I especially like the diagram, it is very clear!

**Comment by uo277776** [comment]
Hi, I made the changes you requested, take a look if anything more has to be change.

Sorry for not doing it sooner, I couldn't do anything all weekend.

**Pull request merged**